README.md

```diff
@@ -1,5 +1,93 @@
+# Awesome Semantic Shapes [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
 
-# Specification 'awesome-semantic-shapes'
+A curated list of Semantic Shapes resources.
 
-This is the repository for awesome-semantic-shapes. You're welcome to contribute! Let's make the Web rock our socks
-off!
+Contributions welcome! Please, read the [Contribution Guidelines](CONTRIBUTING.md) first.
+
+## Contents
+
+- [SHACL Validators](#shacl-validators)
+- [ShEx Validators](#shex-validators)
```

Comment:
Do we need two validator categories?

Reply:
It is useful from "validator user" point of view to separate it. I assume (maybe wrongly), an average project had committed to SHACL or ShEx from the beginning. And project's CTO "selecting from" or "monitoring for a tool switch/upgrade" a particular category.

Reply:
Okay, I realised my mistake. You are right, of course. Makes sense

Reply:
Also, it seems, it will be to long list if we combine it together.